operations/CSM_product_management/Configure_Non_Compute_Nodes.md

```diff
@@ -0,0 +1,168 @@
+# Configure Non-compute Nodes with CFS
+
+Non-compute node (NCN) personalization applies post-boot configuration to the
+HPE Cray EX management nodes. Several HPE Cray EX produce environments outside
+of CSM require NCN personalization to function. Consult the manual for each
+product to configure them on NCNs.
+
+This procedure defines the NCN personalization process for the CSM product using
+the [Configuration Framework Service (CFS)](../configuration_management/Configuration_Management.md).
+
+<a name="set_up_passwordless_ssh"></a>
+## Set Up Passwordless SSH
```

Comment:
Perhaps it is already mentioned elsewhere, but if not...I think we should make clear which steps can be skipped for upgrade scenarios (like this one) and which ones need to be executed each time.

Reply:
The problem with this is that for passwordless SSH, it works out of the box by default. This is only an operational task if the site wants to change the keys to their own. For setting the root password, you have to do it for install only in 1.0, but in later CSM versions, no password update is made unless they add something to vault.